### PR TITLE
Add Humanize v0.5.0 for Julia 0.7, upperbound previous versions.

### PR DESCRIPTION
**Pull request by IainNZ**

### CODE REVIEW

File: Humanize/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.7-beta
+Dates
```
Comment (Keno) [inline Humanize/versions/0.5.0/requires]: stdlibs should not go into REQUIRE. This breaks the old pkg manager.

Comment (IainNZ) [reply Humanize/versions/0.5.0/requires]: Oh interesting. I got complaints I thought from 0.7-beta if I didn't have that in my package's REQUIRE. Should they go in there, but not here?

Comment (Keno) [reply Humanize/versions/0.5.0/requires]: Stdlibs are automatically picked up by the pkg3 system by parsing the source code. You likely needed a `Pkg.resolve` after editing your source code. In any case, putting it here is bad.

Comment (IainNZ) [reply Humanize/versions/0.5.0/requires]: Got it. Will delete this PR and attempt the attobot way for good measure.

**Pull request closed without merging**